### ISSUE TITLE
how to run on windows?

### ISSUE THREAD

**Issue by bewest** [bewest/decoding-carelink#20]
Opening this is a discussion place holder for people wanting to run on windows.

From what I understand things should work on Windows, the trick is:
a.  Getting the OS to create a serial device (it can be observed in the Hardware Manager)
b.  Giving the correct name of the serial device to the scripts.

The name is something like `\\\\USB.\\\MedtronicMMT....` or something like that.
I believe what is happening is that Medtronic's own driver automatically creates the serial device, and puts it in the system registry.

It should be possible to query the Windows registry to find/identify/locate the correct `port` argument, which would allow users to use the auto-scanning feature by default.

Here are some more details:
- https://bitbucket.org/abe_k/arduino-data-logger/src/a57960c71ae52a1873f3ad84fda7820a6757f6af/datalogger/getports/wingetports.py?at=default
- http://stackoverflow.com/questions/12090503/listing-available-com-ports-with-python
- http://stackoverflow.com/questions/11303850/what-is-the-cross-platform-method-of-enumerating-serial-ports-in-python-includi
- https://github.com/makerbot/pyserial/blob/master/serial/tools/list_ports_windows.py -- looks quite promising
##

**Comment by scottleibrand** [comment]
I believe @amazaheri and @bewest got this working.

**Comment by amazaheri** [comment]
indeed, we still need some tweaks to openaps. but we have everything running and validated on Windows 10. As soon as we connect RileyLink, we should either merge or create a new branch ☺
